pinot-controller/src/main/java/org/apache/pinot/controller/ControllerConf.java

```diff
@@ -74,60 +76,115 @@
   }
 
   public static class ControllerPeriodicTasksConf {
+
     // frequency configs
+    @Deprecated
     public static final String RETENTION_MANAGER_FREQUENCY_IN_SECONDS = "controller.retention.frequencyInSeconds";
```

Comment:
I like the idea of adding `DEPRECATED_` prefix to the ones that are about to deprecated in the next release. Maybe we should do that in this PR as well.

Reply:
Done.